src/test/java/io/stargate/sgv2/jsonapi/service/shredding/ShredderTest.java

```diff
@@ -207,6 +209,73 @@ public void shredSimpleWithNumberIdWithTrailingZeroes() {
     }
   }
 
+  @Nested
+  class EJSONDateTime {
+    @Test
+    public void shredDocWithDateTimeColumn() {
+      final long testTimestamp = defaultTestDate().getTime();
```

Comment:
would remove the `defaultTestDate()` and simply use `Instant.now()`.. Make tests more robust as date changes all the time..

Reply:
To me it's exactly the opposite when value changes for any test. I guess matter of preference. :)
(that is, test is not necessarily stable, could theoretically pass or fail depending on luck).

But I can change this along with other changes.